xformers/ops.py

```diff
@@ -171,6 +171,14 @@ def supports(cls, d: "AttentionOpDispatch") -> bool:
             return False
         if d.has_dropout and not cls.SUPPORTS_DROPOUT:
             return False
+        # bfloat16 is only supported on A100+
+        # ... although the kernels can still run and give the
+        # correct result
```

Comment:
Honestly most of the tests passed on P100/V100 for `bf16` (to my surprise!) but a few failed, and I didn't feel like spending time debugging this, so I just disabled bf16 on anything < A100.
We could re-enable them at some point, but it might need some effort.

Reply:
SGTM.

Reply:
@danthe3rd I just bumped into this as I'm trying to use bf16 and I only have V100 GPUs available atm. Do you happen to remember/know what supporting this would take? I'm not really familiar much with CUDA programming but I'm wondering what kind of effort supporting this would require.